core/frontend/src/SheetViewState.ts

```diff
@@ -869,7 +869,7 @@ class RasterAttachment {
   }
 
   public get areAllTileTreesLoaded() {
-    return this._viewport?.view.areAllTileTreesLoaded ?? true;
+    return this._viewport?.view?.areAllTileTreesLoaded ?? true;

```

Comment:
Shouldn't this check `this._viewport.areAllTileTreesLoaded`? The background map might be turned on in the attachment.

Reply:
Yes - I wasn't sure of this one.   Fixed.